tests/Unit/Integrations/IntegrationsLoaderTest.php

```diff
@@ -167,6 +167,7 @@ public function testWeDidNotForgetToRegisterALibraryForAutoLoading()
             $excluded[] = 'slim';
             $excluded[] = 'wordpress';
             $excluded[] = 'yii';
+            $excluded[] = 'nette';
```

Comment:
This list appears to be sorted; can you please stick it between memcached and pdo?

Reply:
Done.